src/static_api.rs

```diff
@@ -93,23 +93,37 @@ impl<'a> Generator<'a> {
                         Bot::Renovate => v1::Bot::Renovate,
                     })
                     .collect(),
-                teams: r
-                    .access
-                    .teams
-                    .iter()
-                    .map(|(name, permission)| {
+                teams: {
+                    let mut teams = Vec::new();
+                    for (team_name, permission) in &r.access.teams {
                         let permission = match permission {
                             RepoPermission::Admin => v1::RepoPermission::Admin,
                             RepoPermission::Write => v1::RepoPermission::Write,
                             RepoPermission::Maintain => v1::RepoPermission::Maintain,
                             RepoPermission::Triage => v1::RepoPermission::Triage,
                         };
-                        v1::RepoTeam {
-                            name: name.clone(),
-                            permission,
+
+                        // Look up the team by name and get all its GitHub teams
+                        let team = self.data.team(team_name).with_context(|| {
+                            format!("failed to find team '{team_name}' in teams directory")
+                        })?;
+                        let github_teams = team.github_teams(self.data).with_context(|| {
+                            format!("failed to get GitHub teams for '{team_name}'")
+                        })?;
+                        for gh_team in github_teams {
```

Comment:
I'm not sure if this is correct/what we want. I think that at the logical level of the REST API, we still want to represent that a specific *team* has access to a repository, without expanding it into GH teams here. We should only do the expansion in `sync-team` where we convert this logical representation into actual GH permissions.

Of course this will require changes in sync-team, in particular around `calculate_permission_diffs`. We should then also explicitly represent both the org name and the team name in the diffs.

Reply:
I marked this as resolved after a discussion I had with Kobzol.
We both agree it's better to keep things as they are to avoid making breaking changes to the API.